src/ui/views/login/content.ts

```diff
@@ -0,0 +1,35 @@
+import {getContent} from '../content-base';
+
+const title = 'Login';
+
+const body = `
+<p>
+  Cycode extension requires pre-installed
+  <a
+    class="styled-link"
+    href="https://github.com/cycodehq/cycode-cli#install-cycode-cli"
+  >Cycode CLI</a>
+</p>
+<button id="authenticate-button">Authenticate</button>
+
+<p>
+  To learn more about how to use Cycode in VSCode
+  <a
+    class="styled-link"
+    href="https://github.com/cycodehq/vscode-extension/blob/main/README.md"
+    >read our docs.</a>
```

Comment:
To be consistent with src/ui/views/main/content.ts 

```suggestion
    >read our docs</a>.
```